SmartHouse.Xamarin/SmartHouse.Xamarin.Core/DAL/SmartHouseClient.cs

```
@@ -0,0 +1,33 @@
+﻿using MvvmCross.Plugin.Network.Rest;
+using System;
+using System.Threading.Tasks;
+
+namespace SmartHouse.Xamarin.Core.DAL
+{
+    public class SmartHouseClient<T> : ISmartHouseClient<T> where T : class
+    {
+        private IMvxJsonRestClient _client { get; set; }
+
+        public SmartHouseClient(IMvxJsonRestClient client)
+        {
+            _client = client;
+        }
+
+        public Task<MvxDecodedRestResponse<V>> MakeRequestForAsync<V>(MvxJsonRestRequest<T> request)
+        {
+            try
+            {
+                return _client.MakeRequestForAsync<V>(request);
+            }
+            catch (TimeoutException)
+            {
```

do not leave empty catches. at least log errors